src/tests/simulation/src/slt.rs

```diff
@@ -112,7 +113,13 @@ pub async fn run_slt_task(cluster: Arc<Cluster>, glob: &str, opts: &KillOpts) {
 
             // For normal records.
             if !kill {
-                match tester.run_async(record).await {
+                match tester
+                    .run_async(record.clone())
+                    .timed(|_res, elapsed| {
+                        println!("Record {:?} finished in {:?}", record, elapsed)
+                    })
+                    .await
+                {
```

Comment:
I came up with a simpler solution that is more specialized and can avoid TimedFuture utility.
But the current way also looks good to me for its universality.

```suggestion
                match run_record(&mut tester, &record).await {
```

if we have:
```rust
async fn run_record(
    tester: &mut sqllogictest::Runner<RisingWave>,
    record: &sqllogictest::Record,
) -> Result<(), sqllogictest::TestError> {
    let t0 = std::time::Instant::now();
    let res = tester.run_async(record.clone()).await;
    println!("Record {:?} finished in {:?}", record, t0.elapsed());
    res
}
```